include/RED4ext/Scripting/Natives/entAnimationControllerComponent.hpp

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#include <RED4ext/Common.hpp>
+#include <RED4ext/Handle.hpp>
+#include <RED4ext/NativeTypes.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/AnimDatabaseCollection.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/IKTargetParams_Update.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/IKTargetRef.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/LookAtParams_UpdatePositions.hpp>
+#include <RED4ext/Scripting/Natives/Generated/anim/LookAtRef.hpp>
+#include <RED4ext/Scripting/Natives/Generated/ent/IComponent.hpp>
+#include <RED4ext/Scripting/Natives/Generated/ent/IOrientationProvider.hpp>
+#include <RED4ext/Scripting/Natives/Generated/ent/IPositionProvider.hpp>
+#include <cstdint>
+
+namespace RED4ext
+{
+namespace anim
+{
+struct ActionAnimDatabase;
+}
+namespace ent
+{
+struct AnimationControlBinding;
+struct AnimationControllerComponent;
+
+struct IKTargetController
+{
+    struct IKTargetData
+    {
+        anim::IKTargetRef ikTargetref;                    // 00
+        Handle<IPositionProvider> positionProvider;       // 10
+        Handle<IOrientationProvider> orientationProvider; // 20
+    };
+    RED4EXT_ASSERT_SIZE(IKTargetData, 0x30);
+
+    AnimationControllerComponent& animationControllerComponent; // 00
+    uint64_t unk08;                                             // 08
+    DynArray<IKTargetData> IKTargetData;                        // 10
+    Handle<anim::IKTargetParams_Update> IKparams;               // 20
+};
+RED4EXT_ASSERT_SIZE(IKTargetController, 0x30);
+
+struct LookAtController
+{
+    struct LookatData
+    {
+        anim::LookAtRef lookAtRef;                  // 00
+        Handle<IPositionProvider> positionProvider; // 10
+        uint32_t unk20;                             // 20
+    };
+    RED4EXT_ASSERT_SIZE(LookatData, 0x28);
+    struct AdditionalLookatData
+    {
+        CName partName;                       // 00
+        DynArray<anim::LookAtRef> lookAtRefs; // 08
+    };
+    RED4EXT_ASSERT_SIZE(AdditionalLookatData, 0x28);
+
+    AnimationControllerComponent& animationControllerComponent; // 00
+    uint32_t unkCounter;                                        // 08
```

Comment:
Naming: `unk08` or `counter`
You can add a comment if you know something about the counter.